src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
 ---
-description: Parent to child communication
+description: The component could expose property attributes to receive data from the parent html component
 ---
 
 # Properties
 
-Properties enable child and parent components to communicate with each other.
+It is a common need to receive data from the parent component using properties.
 
-## Derive macro
+The properties are defined in a dedicated Rust struct derivating `Properties` and `Clone`traits.
 
-Don't try to implement `Properties` yourself, derive it by using `#[derive(Properties)]` instead.
+> This struct is usually named `Props`
 
-{% hint style="info" %}
-Types for which you derive `Properties` must also implement `Clone`. This can be done by either using `#[derive(Properties, Clone)` or manually implementing `Clone` for your type.
-{% endhint %}
+The properties bag is attached to the state adding the field `props` in the component struct and defining the `Properties` `Type` in the component trait implementation.
 
-### Required attributes
+The properties have to be intialized in the `create` method. As for `link`, the framework push to the method the bag of the properties and can be used to attach it to the internal state.
 
-The fields within a struct that derives `Properties` are required by default. When a field is missing and the component is created in the `html!` macro, a compiler error is returned. For fields with optional properties, use the `#[prop_or_default]` attribute to use the default value for that type when the prop is not specified. To specify a value, use the `#[prop_or_else(value)]` attribute where value is the default value for the property. For example, to default a boolean value as `true`, use the attribute `#[prop_or_else(true)]`. It is common for optional properties to use the `Option` enum which has the default value `None`.
+It' possible to define some attributes to the properties as:
```

Comment:
"It'" => `it's`/`it is`

Reply:
Thanks

Anyway I've change the sentence to:
***
Properties may be defined as:
***

It fit better with the next sentences.

Do not heistate to comment in case you disagree